backoffice/src/Models/ViewModels/WreckMaterialView.cs

```diff
@@ -27,12 +27,14 @@ public WreckMaterialView(WreckMaterial wreckMaterial)
         WhereSecured = wreckMaterial.WhereSecured;
         Created = wreckMaterial.Created;
         LastModified = wreckMaterial.LastModified;
+        Address = new AddressView(wreckMaterial.Address);
     }
 
 
     public Guid Id { get; }
     public Guid DroitId { get; }
     public string Name { get; } = string.Empty;
+    public AddressView Address { get; } = new();
```

Comment:
Can we rename this StorageAddress pls?
public AddressView StorageAddress { get; } = new();